{"timestamp": "2025-03-26 20:27:04,095", "level": "INFO", "logger_name": "werkzeug", "message": "\u001b[31m\u001b[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.\u001b[0m\n * Running on http://127.0.0.1:5000", "thread_name": "MainThread"}
{"timestamp": "2025-03-26 20:27:04,095", "level": "INFO", "logger_name": "werkzeug", "message": "\u001b[33mPress CTRL+C to quit\u001b[0m", "thread_name": "MainThread"}
{"timestamp": "2025-03-26 20:27:09,626", "level": "INFO", "logger_name": "__main__", "message": "Test user logging", "thread_name": "Thread-1 (process_request_thread)", "user": "john_doe", "ip": "192.168.1.1", "tags": ["auth", "success"], "traceId": "fbc57da0e837fc5180ca663ac0f3a7c4", "spanId": "872bd28ab41c0b7a"}
{"timestamp": "2025-03-26 20:27:09,626", "level": "WARNING", "logger_name": "__main__", "message": "Some warn", "thread_name": "Thread-1 (process_request_thread)", "traceId": "fbc57da0e837fc5180ca663ac0f3a7c4", "spanId": "872bd28ab41c0b7a"}
{"timestamp": "2025-03-26 20:27:09,626", "level": "ERROR", "logger_name": "__main__", "message": "Some error", "thread_name": "Thread-1 (process_request_thread)", "traceId": "fbc57da0e837fc5180ca663ac0f3a7c4", "spanId": "872bd28ab41c0b7a"}
{"timestamp": "2025-03-26 20:27:09,626", "level": "ERROR", "logger_name": "__main__", "message": "Database error", "thread_name": "Thread-1 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"C:\\Users\\FedosPopados\\PycharmProjects\\todo\\app\\app.py\", line 34, in LogPyController\n    raise Exception(\"Test exception\")\nException: Test exception", "component": "database", "query": "SELECT * FROM users", "traceId": "fbc57da0e837fc5180ca663ac0f3a7c4", "spanId": "872bd28ab41c0b7a"}
{"timestamp": "2025-03-26 20:27:09,628", "level": "INFO", "logger_name": "werkzeug", "message": "127.0.0.1 - - [26/Mar/2025 20:27:09] \"GET /api/logs HTTP/1.1\" 200 -", "thread_name": "Thread-1 (process_request_thread)"}
{"timestamp": "2025-03-26 21:15:40,508", "level": "INFO", "logger_name": "werkzeug", "message": "\u001b[31m\u001b[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.\u001b[0m\n * Running on http://127.0.0.1:5000", "thread_name": "MainThread"}
{"timestamp": "2025-03-26 21:15:40,509", "level": "INFO", "logger_name": "werkzeug", "message": "\u001b[33mPress CTRL+C to quit\u001b[0m", "thread_name": "MainThread"}
{"timestamp": "2025-03-26 21:15:45,717", "level": "INFO", "logger_name": "__main__", "message": "Test user logging", "thread_name": "Thread-1 (process_request_thread)", "user": "john_doe", "ip": "192.168.1.1", "tags": ["auth", "success"], "traceId": "190a9106d442a26cffb13115b2ae14e5", "spanId": "e29800cc29a6c0bb"}
{"timestamp": "2025-03-26 21:15:45,717", "level": "WARNING", "logger_name": "__main__", "message": "Some warn", "thread_name": "Thread-1 (process_request_thread)", "traceId": "190a9106d442a26cffb13115b2ae14e5", "spanId": "e29800cc29a6c0bb"}
{"timestamp": "2025-03-26 21:15:45,717", "level": "ERROR", "logger_name": "__main__", "message": "Some error", "thread_name": "Thread-1 (process_request_thread)", "traceId": "190a9106d442a26cffb13115b2ae14e5", "spanId": "e29800cc29a6c0bb"}
{"timestamp": "2025-03-26 21:15:45,717", "level": "ERROR", "logger_name": "__main__", "message": "Database error", "thread_name": "Thread-1 (process_request_thread)", "exc_info": "Traceback (most recent call last):\n  File \"C:\\Users\\FedosPopados\\PycharmProjects\\todo\\app\\app.py\", line 34, in LogPyController\n    raise Exception(\"Test exception\")\nException: Test exception", "component": "database", "query": "SELECT * FROM users", "traceId": "190a9106d442a26cffb13115b2ae14e5", "spanId": "e29800cc29a6c0bb"}
{"timestamp": "2025-03-26 21:15:45,718", "level": "INFO", "logger_name": "werkzeug", "message": "127.0.0.1 - - [26/Mar/2025 21:15:45] \"GET /api/logs HTTP/1.1\" 200 -", "thread_name": "Thread-1 (process_request_thread)"}
